bc_obps/registration/tests/endpoints/v2/test_operators_endpoint.py

```diff
@@ -0,0 +1,104 @@
+        page_1_response_id = response_data[0].get('id')
+        assert len(response_data) == PAGE_SIZE
+        # Get the page 2 response
+        response = TestUtils.mock_get_with_auth_role(
+            self,
+            "cas_admin",
+            custom_reverse_lazy('list_operators_v2') + "?page=2&sort_field=created_at&sort_order=desc",
+        )
+        assert response.status_code == 200
+        response_data = response.json().get('data')
+        # save the id of the first paginated response item
+        page_2_response_id = response_data[0].get('id')
+        assert len(response_data) == PAGE_SIZE
+        # assert that the first item in the page 1 response is not the same as the first item in the page 2 response
+        assert page_1_response_id != page_2_response_id
+
+        # Get the page 2 response but with a different sort order
+        response = TestUtils.mock_get_with_auth_role(
+            self,
+            "cas_admin",
+            custom_reverse_lazy('list_operators_v2') + "?page=2&sort_field=created_at&sort_order=asc",
+        )
+        assert response.status_code == 200
+        response_data = response.json().get('data')
+        # save the id of the first paginated response item
+        page_2_response_id_reverse = response_data[0].get('id')
+        assert len(response_data) == PAGE_SIZE
+        # assert that the first item in the page 2 response is not the same as the first item in the page 2 response with reversed order
+        assert page_2_response_id != page_2_response_id_reverse
+
+    def test_operators_endpoint_list_operators_v2_with_filter(self):
+        operator_baker({"legal_name": 'gouda'})
+        operator_baker({"cra_business_number": '118675309'})
+        for i in range(60):
+            operator_baker()
+
+        # Get the default page 1 response
+        response = TestUtils.mock_get_with_auth_role(
+            self, "cas_admin", custom_reverse_lazy('list_operators_v2') + "?legal_name=gouda"
+        )
+        assert response.status_code == 200
+        response_data = response.json().get('data')
+        assert len(response_data) == 1
+        for item in response_data:
+            assert item.get('legal_name') == 'gouda'
+
+        # Test with a status filter that doesn't exist
+        response = TestUtils.mock_get_with_auth_role(
+            self, "cas_admin", custom_reverse_lazy('list_operators_v2') + "?legal_name=havarti"
+        )
+        assert response.status_code == 200
+        response_data = response.json().get('data')
+        assert len(response_data) == 0
+
+        # Test with a cra_business_number filter
+        response = TestUtils.mock_get_with_auth_role(
+            self, "cas_admin", custom_reverse_lazy('list_operators_v2') + "?cra_business_number=118675309"
+        )
+        assert response.status_code == 200
+        response_data = response.json().get('data')
+        assert len(response_data) == 1
```

Comment:
I suggest validating the `cra_business_number` in the response to ensure it aligns with our expectations.